cocos/core/director.ts

```diff
@@ -297,21 +288,6 @@ export class Director extends EventTarget {
         legacyCC.game.once(Game.EVENT_RENDERER_INITED, this._initOnRendererInitialized, this);
     }
 
-    /**
-     * @en Calculates delta time since last time it was called, the result is saved to an internal property.
-     * @zh 计算从上一帧到现在的时间间隔，结果保存在私有属性中
-     */
-    public calculateDeltaTime (now) {
-        if (!now) now = performance.now();
-
-        this._deltaTime = now > this._lastUpdate ? (now - this._lastUpdate) / 1000 : 0;
-        if (DEBUG && (this._deltaTime > 1)) {
-            this._deltaTime = 1 / 60.0;
-        }
-
-        this._lastUpdate = now;
-    }
-
```

Comment:
I think the public method can't be removed, should be marked as deprecation instead

or it would be a breaking change in v3.3
maybe can call `game._calculateDT()` here

Reply:
It can still be invoked, I have added removed warning through `removeProperty` in deprecated